test/FunctionalTests/Grpc.AspNetCore.FunctionalTests.csproj

```diff
@@ -47,6 +47,10 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
 
+    <None Update="localhost.pfx">
```

Comment:
what's the expiration on this? is it going to bite us in a year? or is it a century?

Reply:
I set it to 10 years.